Remove | tee $PIPE and systemd-cat

wesbarnett opened the pull request:
Remove duplication of logs.

Fixes #101.

wesbarnett committed:
Change | tee $PIPE to > $PIPE

Remove duplication of logs.

Fixes #101.

NicoHood commented:
Well, this looks trivial. Why was $PIPE even used?

I found this:
https://github.com/wesbarnett/snap-sync/commit/e7085c4f92956290638739677e03837d144781dd

I am not sure what the issue is here, is it only to print to stderr?
https://stackoverflow.com/questions/10963653/what-method-should-i-use-to-write-error-messages-to-stderr-using-printf-in-a

wesbarnett commented:
They problem I was trying to solve was getting the output to print both to the screen and to the systemd journal, since this can be set to run with a systemd unit.

This sets up $PIPE here:
https://github.com/wesbarnett/snap-sync/blob/master/bin/snap-sync#L41-L43

But, after some searching I think the correct way is to print to stdout/stderr without setting up systemd-cat and then in the systemd unit specify `StandardOutput=journal+console`

wesbarnett commented:
Now it looks like it's not even necessary any more. It looks like the stdout automatically goes to the systemd journal for the unit

wesbarnett committed:
Remove unneeded systemd-cat

If this is run by a systemd service, StandardOutput defaults to
journal already, so no need to control that here.

NicoHood commented:
That is weird. But didnt you introduce that feature because it did not work before? Why does it work now?

Looks way cleaner though :-)
